curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-67-maximum-path-sum-ii.english.md

```diff
@@ -7,13 +7,22 @@ forumTopicId: 302179
 
 ## Description
 <section id='description'>
+
 By starting at the top of the triangle below and moving to adjacent numbers on the row below, the maximum total from top to bottom is 23.
-37 4
-2 4 6
-8 5 9 3
+
+<div style='text-align: center;'>
+  <strong><span style='color: red;'>3</span></strong><br>
+  <strong><span style='color: red;'>7</span></strong> 4<br>
+  2 <strong><span style='color: red;'>4</span></strong> 6<br>
+  8 5 <strong><span style='color: red;'>9</span></strong> 3
+</div>
+
 That is, 3 + 7 + 4 + 9 = 23.
+
 Find the maximum total from top to bottom in triangle.txt (right click and 'Save Link/Target As...'), a 15K text file containing a triangle with one-hundred rows.
```

Comment:
We're not currently hosting `triangle.txt`.  Should we?

Reply:
No, thank you for pointing this out. I might have messed this up with a force push somewhere along the line. I'll recreate an array with these numbers and change the text here.